nemo/collections/asr/parts/numba/rnnt_loss/utils/cuda_utils/gpu_rnnt_kernel.py

```diff
@@ -875,3 +877,500 @@ def compute_multiblank_grad_kernel(
             # update internal index through the thread_buffer;
             # until idx < V + 1, such that entire vocabulary has been updated.
             idx += GPU_RNNT_THREAD_SIZE
+
+
+@cuda.jit(device=True, inline=True)
```

Comment:
Could you move this function to the top of the file - closer to logp() kernel?

Reply:
++

Reply:
Done.